pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -1510,6 +1513,8 @@
     private String managedLedgerDataReadPriority = OffloadPolicies.OffloadedReadPriority.TIERED_STORAGE_FIRST
             .getValue();
 
+    private long stickConsumerBackOffDelayTimeMs = 500;
```

Comment:
Seems not related to this PR.

Reply:
I add this for config backoff time for pr 10096.